metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_utils.py

```diff
@@ -79,12 +80,25 @@ def create_snowsight_base_url(
         cloud: str,
         privatelink: bool = False,
     ) -> Optional[str]:
+        if cloud:
+            url_cloud_provider_suffix = f".{cloud}"
+
+        if cloud == SnowflakeCloudProvider.AWS:
+            if cloud_region_id in [
```

Comment:
Can we please extract these regions list as a constant and mark this constant with the snowflake link -> from where we figured out these ?